_i18n/ml/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "AVR, ARM അടിസ്ഥാനമാക്കിയുള്ള കീബോർഡുകൾക്ക് വേണ്ടിയുള്ള ഓപ്പൺ സോഴ്സ് ഫേംവെയർ"
+permalink: /ml/
+lang: ml
+---
+Atmel AVR നും ARM കൺട്രോളറിനും അഥവാ [OLKB product line](http://olkb.com), [ErgoDox EZ](http://www.ergodox-ez.com) കീബോർഡ്, [Clueboard product line](http://clueboard.co/) തുടങ്ങിയവയ്ക്ക് ഉപകാരപ്രദമായ കുറേ സവിശേഷതകളുള്ള, 
+[tmk\_keyboard firmware](http://github.com/tmk/tmk_keyboard) നെ അടിസ്ഥാനമാക്കിയുള്ള കീബോർഡ് ഫേംവെയറാണ് ഇത്.
```

Comment:
Lines 6 and 7 seem to be in reverse order of how they are on the English site.

Reply:
Hi @noroadsleft , the reverse order is to get the meaningful sentence without any redundant words. If we convert it as it is to Malayalam, there will be some unnecessary words which will confuse people.

Reply:
@sagarvd01 Thanks for the explanation. Obviously I don't know Malayalam, but it just looked weird to me.